pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -529,6 +529,28 @@ public void testBase64Func()
       assertEquals(encoded, toBase64(toUtf8(original)));
       assertEquals(decoded, fromUtf8(fromBase64(toBase64(toUtf8(original)))));
     }
+
+    // Test select with group by order by limit
+    sqlQuery = "SELECT toBase64(toUtf8(AirlineID)) "
+        + "FROM mytable "
+        + "GROUP BY toBase64(toUtf8(AirlineID)) "
+        + "ORDER BY toBase64(toUtf8(AirlineID)) DESC "
+        + "LIMIT 10";
```

Comment:
there's a problem with this. 
```
SELECT toBase64(toUtf8(AirlineID)) AS AirlineID
FROM mytable
GROUP BY toBase64(toUtf8(AirlineID))
ORDER BY toBase64(toUtf8(AirlineID)) DESC 
LIMIT 10
```
will failed b/c of the reference to `AirlineID` is both in the function `toBase64 and `AS` aliasing 
did this query ever succeed before the expand identifier?